LayerClient/LayerClient.py

```diff
@@ -1,4 +1,6 @@
 # -*- coding: utf-8 -*-
+from __future__ import absolute_import, unicode_literals
```

Comment:
We are working on 2.7 but as I understood only version 3 has absolute imports mandatory. (http://stackoverflow.com/a/11741846/2249584)